app/models/manageiq/providers/foreman/provider.rb

```diff
@@ -74,4 +74,5 @@ def ensure_managers
   def self.refresh_ems(provider_ids)
     EmsRefresh.queue_refresh(Array.wrap(provider_ids).collect { |id| [base_class, id] })
   end
+  private_class_method :refresh_ems
```

Comment:
@bdunne does this look ok? I don't see anything calling self.class.refresh_ems or similar, which would indicate it's truly not private.